drivers/storage/portworx/util/util.go

```diff
@@ -1471,19 +1454,18 @@ func CountStorageNodes(
 			if !isDRSetup {
 				storageNodesCount++
 			} else {
-				if _, ok := k8sNodesStoragePodCouldRun[node.SchedulerNodeName]; ok {
+				if node.ClusterDomain == currentClusterDomain {
 					storageNodesCount++
 				} else {
-					logrus.Debugf("node %s should not run portworx", node.SchedulerNodeName)
+					logrus.Debugf("node %s is not part of cluster domain %s ", node.SchedulerNodeName, currentClusterDomain)
```

Comment:
nit: Let's print the node_name/node_id as part of the output in case the scheduler node name is not present.